docs/Prelude.md

```diff
@@ -983,4 +983,20 @@ instance showArray :: (Show a) => Show (Array a)
 instance showOrdering :: Show Ordering
 ```
 
+#### `NaturalTransformation`
+
+``` purescript
+type NaturalTransformation f g = forall a. f a -> g a
+```
+
+A natural transformation
+
+#### `Natural`
+
+``` purescript
+type Natural f g = NaturalTransformation f g
```

Comment:
One advantage of adding this to the Prelude is to increase the Prelude's didactic function. When starting out with Purescript, I found it exceedingly instructive to simply sit down and read through the Prelude.